test/base/autorunAsync.js

```diff
@@ -130,6 +130,94 @@ test("autorunAsync passes Reaction as an argument to view function", function(do
     }, 1000)
 })
 
+test("autorunAsync accepts a scheduling function", function(done) {
+    var a = m.observable({
+        x: 0,
+        y: 1
+    })
+
+    var autoRunsCalled = 0
+    var schedulingsCalled = 0
+
+    debugger
```

Comment:
I don't think this is supposed to be here?

Reply:
Correct, removed, thanks!